data/mappings/keyboard_aliases.hjson

```diff
@@ -927,6 +927,9 @@
     "kprepublic/bm68hsrgb": {
         "target": "kprepublic/bm68hsrgb/rev1"
     },
+    "laser_ninja/pumpkin_pad": {
+        "target": "laser_ninja/pumpkinpad"
+    },
```

Comment:
Needs to be further up, above `lattice60`. This section is for boards moved in 2022Q1.